windows-agent/internal/proservices/registrywatcher/watcher.go

```diff
@@ -31,7 +30,7 @@ type Service struct {
 	db       *database.DistroDB
 }
 
-var registryPath = []string{"Software", "Canonical", "UbuntuPro"}
+const registryPath = `Software\Canonical\UbuntuPro`
```

Comment:
hum, duplication with validPaths, but, ok, you can’t reuse it without exposing it…

Reply:
Yeah, I did not want to expose it.